src/sous_chef/templates/base.html

```diff
@@ -70,6 +70,12 @@
     {% endif %}
     <!-- WARNING :  Do not move this line before a manual import of script! -->
     {% block extrajs %}{% endblock %}
+
+    <script type="application/javascript">
+      $(function () {
+        for(var f=document.forms,i=f.length;i--;)f[i].setAttribute("novalidate",i)
+      });  // disable HTML5 form validation
+    </script>
```

Comment:
@lingxiaoyang In most parts the html5 validation work just fine, the only issue is with the fields hidden by semantic-ui and replaced with a most "nice" look. I'm ok with removing the html5 validation from this hidden fields, but remove all client validation seems unnecessary to me and will affect the performance of the application, we don't need to hit the server when we already know that we have missing information or validations problems.